test/e2e/network/service.go

```diff
@@ -1339,8 +1339,15 @@ var _ = SIGDescribe("Services", func() {
 
 	It("should be able to create an internal type load balancer [Slow]", func() {
 		framework.SkipUnlessProviderIs("azure", "gke", "gce")
+		if framework.ProviderIs("gke", "gce") {
+			framework.SkipUnlessNodeCountIsAtMost(framework.GCPMaxInstancesInInstanceGroup)
```

Comment:
Btw - instead of totally disabling the test, can we create an LB but with only GCPMaxInstancesInInstanceGroup instances in the target pool?

Reply:
*InstanceGroup.

It could be done, but it would need to be within service/ingress controllers. However, you would then run into other issues. ESIPP would not function properly because a subset of your pods could be on the ignored instances.  Non-ESIPP load balancers would work but the load would be imbalanced.

Reply:
Sounds reasonable. What does ESIPP stand for btw? :)

Reply:
ESIPP=External Source IP Preservation :)